features/facebookImport/src/views/report/reportWrapper.jsx

```diff
@@ -23,10 +25,14 @@ const ReportWrapper = () => {
     const activeReportStories = computedReportStoriesList.filter(
         (reportStory) => reportStory.active
     );
-    const statusReport = new MinistoriesStatusReport([
-        ...computedReportStoriesList,
-        ...computedMinistories,
-    ]);
+    const statusReport = new MinistoriesStatusReport({
+        ministories: [...computedReportStoriesList, ...computedMinistories],
```

Comment:
There might be some problem here, I think, at least that's my hunch based on the error, which reveals problems with `this._ministoriesData`. If it's not this argument, it might be related to this initialization. Can you please check?

Reply:
I can't see how this is wrong, I might be missing something. Maybe it's another call?

Reply:
It's just that we changed the class constructor for MinistoriesStatusReport and now it takes an object as argument rather than just the array. Just need to update that in the testing as well :)